docs/_sources/getting_started_tutorial.md.txt

```diff
@@ -0,0 +1,231 @@
+necessary file paths in a `payload`. We can also specify a common prefix to all paths in a `payload` in a `path` argument.
+```python
+path = "model/"
+payload = ['src/func_main.py']
+```
+
+Hydrosphere Serving has a strictly typed inference engine, so before uploading our model we need to specify it's signature with
+`SignatureBuilder`. Signature contains information about which method inside your `func_main.py` should be called,
+ as well as what its inputs and outputs shapes and types are.
+ 
+```python
+from hydrosdk.contract import SignatureBuilder, ModelContract
+
+signature = SignatureBuilder('infer') \
+                .with_input('x', 'double', "scalar") \
+                .with_output('y', 'double', "scalar").build()
+
+contract = ModelContract(predict=signature)
+```
+
+At this point we can combine all our efforts into the LocalModel object. LocalModels are models before they get
+ uploaded to the cluster. LocalModels are containers for all the information required to instantiate a ModelVersion
+  in a Hydrosphere cluster. We'll call this model `"sqrt_model"`. 
+  
+
+Moreover, we need to specify environment in which our model will run.
+Such environments are called Runtimes. You can learn more about them [here](https://hydrosphere.io/serving-docs/latest/overview/concepts.html#runtimes).
+In this tutorial we will use default Python 3.7 runtime.
+This runtime uses `src/func_main.py` script as an entry point, that's why we organised our files as we did.
+
+```python
+from hydrosdk.modelversion import LocalModel
+from hydrosdk.image import DockerImage
+
+sqrt_local_model = LocalModel(name="sqrt_model",
+                              contract=contract,
+                              runtime=DockerImage("hydrosphere/serving-runtime-python-3.7", "2.3.2", None),
+                              payload=payload,
+                              path=path)
+```
+
+After packing all necessary information into a LocalModel, we finally can upload it.
+```python
+upload_response = sqrt_local_model.upload(cluster, wait=True)
+```
+
+Let's check whether our model was successfully uploaded to the platform by looking for it.
+```python
+from hydrosdk.modelversion import ModelVersion
+sqrt_model = ModelVersion.find(cluster, name="sqrt_model", version=1)
+```
+
+We are finished with uploading our model. Now we can get to the part where we develop a client code for our model. 
+
+## Connect to your deployed model
+ 
+We have uploaded our model - it's stored and versioned, but it's not running yet - we need to deploy it. To deploy a model
+you should create an Application - linear pipeline of ModelVersions with monitoring and other benefits. 
+You can learn more about Applications [here](https://hydrosphere.io/serving-docs/latest/overview/concepts.html#applications).
+
+For the sake of simplicity, we'll create just a Servable - a bare deployed instance of our model version without any benefits. 
```

Comment:
Servables are mainly used within the platform and there's no need to expose them to the end user. Instead, we should just focus on a single-staged applicaiton in this example, as it will be helpful further down the road as well.